src/FormBuilder/ViewModels/FormBuilderViewModel.php

```diff
@@ -110,6 +113,25 @@ public function getGoalTypeOption(
         ];
     }
 
+    /**
+     * @unreleased
+     */
+    public function getGoalProgressOption(
+        string $value,
+        string $label,
+        string $description,
+        bool $isCustom = false
```

Comment:
The `isCustom` flag seems redundant here. Since there are only two options, should we instead just do comparisons against the option name/value?

Reply:
Yes, you are right. I'll simplify this.

Reply:
Resolved: https://github.com/impress-org/givewp/pull/7368/commits/abeee7fb7856a00517b5b23d0c8c3be450e62ad0